frontera/contrib/scrapy/converters.py

```diff
@@ -26,7 +28,7 @@ def to_frontier(self, scrapy_request):
         if callable(eb):
             eb = _find_method(self.spider, eb)
 
-        scrapy_meta = scrapy_request.meta
+        scrapy_meta = dict_to_bytes(scrapy_request.meta, keys_only=True)
```

Comment:
Is this line really required? My thinking is that since this code is in `to_frontier`, scrapy's meta will be in the original form as scrapy generated it.